change/@fluentui-make-styles-20c29498-cb40-4e3e-8607-074e0283eb81.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Update jest-environment comment formatting to fix lint error",
+  "packageName": "@fluentui/make-styles",
+  "email": "behowell@microsoft.com",
+  "dependentChangeType": "patch"
```

These should be type none since it's a test-only change.
```suggestion
  "type": "none",
  "comment": "Update jest-environment comment formatting to fix lint error",
  "packageName": "@fluentui/make-styles",
  "email": "behowell@microsoft.com",
  "dependentChangeType": "none"
```